email_totals/ses.py

```diff
@@ -346,24 +346,37 @@ def build_unowned_email_body(unowned_data, account_names):
     return html_body, text_body
 
 
+def add_cc_list(primary):
+    """
+    Add the CC addresses to the list of recipients, if any
+    """
+    recipients = [primary, ]
+    cc_list = os.environ['CC_LIST'].split(',')
+    if cc_list != ['']:
+        recipients.extend(cc_list)
+    return recipients
+
+
 def send_report_email(recipient, body_html, body_text, period):
     """
     Send a per-user report email
     """
     subject = f"AWS Monthly Cost Report ({period})"
-    send_email(recipient, subject, body_html, body_text)
+    recipients = add_cc_list(recipient)
+    send_email(recipients, subject, body_html, body_text)
 
 
 def send_unowned_email(body_html, body_text, period):
     """
     Send a report on unowned costs to the admin recipient
     """
-    admin = os.environ['ADMIN_EMAIL']
     subject = f"AWS Unowned Costs ({period})"
-    send_email(admin, subject, body_html, body_text)
+    admin = os.environ['ADMIN_EMAIL']
+    recipients = add_cc_list(admin)
```

Comment:
if i understand correctly, the idea is to setup an email list (google group) then allow users to subscribe to that group correct?  If that's the case then does `admin` need to be here anymore?  I mean can't admins just subscribe to the google group instead?

Reply:
I think the admin address is still useful, it's specifically for the unowned resources report which doesn't have a tagged owner for the TO field. the new google group will also receive copies of all of the user reports sent to tagged owners, and I personally would prefer to only get the admin report.

Reply:
maybe i don't understand.  if i am an admin and i also join the CC alias wouldn't i get duplicate emails? or is that just not a valid use case?

Reply:
if you are an admin and join the CC group, you would get duplicate copies of the admin reports as well as copies of every individual user report

Reply:
do you not see that being a problem @jesusaurus ?  maybe we can define groups where we don't get duplicate notifications or maybe we can dedupe the list of recipients?

Reply:
@zaro0508 yeah I agree it's not ideal, but I don't see how it's possible to create a new google group that will receive a copy of every email report without subscribers to that google group getting a duplicate copy of any email reports they were already receiving previously. if a tagged owner subscribes to the CC group, I can't think of a way to prevent the owner from receiving two copies of the email, one sent directly to the owner email and another to the CC group. and the only ways I can think of to avoid duplicate admin reports to someone in both the admin group and the CC group is to either not send admin reports to the CC group (which would mean that LT members signing up for CC group will not see the untagged resources report) or to combine both into a single group (which would mean that cloud admins can't get the admin reports without also getting a copy of every owner report).

can you suggest a way to approach the request that won't send duplicate reports?

Reply:
Please remember that the primary use case is the president (who is not a user) wishing to have visibility into reports being sent to users.  The case of a user (who already receives a report) also adding themselves as a 'cc' is an edge case that doesn't demand a lot of consideration, in my opinion.